File: CS/CustomMessaging.Module.Win/WinModule.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;

using DevExpress.ExpressApp;

namespace CustomMessaging.Module.Win {
    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class CustomMessagingWindowsFormsModule : ModuleBase {
        public CustomMessagingWindowsFormsModule() {
            InitializeComponent();
        }
    }
}

File: VB/CustomMessaging.Module.Win/WinModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.ComponentModel

Imports DevExpress.ExpressApp

Namespace CustomMessaging.Module.Win
    <ToolboxItemFilter("Xaf.Platform.Win")> _
    Public NotInheritable Partial Class CustomMessagingWindowsFormsModule
        Inherits ModuleBase
        Public Sub New()
            InitializeComponent()
        End Sub
    End Class
End Namespace
